index.js

```diff
@@ -34,6 +49,18 @@
 
 
 // ***** Question 4 *****
+function sameSameButDifferent(num, maybeNum) {
+  if (num === maybeNum) {
+    return "same same"
+  }
+  else if (num == maybeNum) {
+    return "same same (but different)"
+  }
+  else {
+    return "different"
+  }
+  
+}
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 4", "color: red") 
```

Comment:
Nice! Another solution would be to convert the string to a number with parseInt to avoid `==` :

`} else if (num === parseInt(maybeNum)) {`